projects/kalidea/kaligraphi/src/lib/molecules/kal-tabs/README.md

```diff
@@ -1,10 +1,10 @@
-# TAB PANEL
+#TAB PANEL
```

Comment:
do not modify other files than your feature

Reply:
Just this readme in PR